app/code/Magento/InventoryInStorePickupAdminUi/view/adminhtml/templates/order/create/shipping/method/sources_form.phtml

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\InventoryInStorePickupAdminUi\ViewModel\CreateOrder\SourcesForm;
+
+// @codingStandardsIgnoreFile
+/** @var SourcesForm $viewModel */
+$viewModel = $block->getData('view_model');
```

Comment:
Let's go with the conventional `$block->getViewModel()`

Reply:
Just don`t like magic. Fixed.